### DISCUSSION TITLE
Are we using IRC actively?

### DISCUSSION THREAD

**Discussion by rohitgeddam** [Homebrew/discussions#86] [Everyday usage]
Are we still using IRC? when i logged in there was no one there.

**Comment by SMillerDev** [comment]
As far as I know homebrew doesn't have an IRC channel. Where did you find this channel?

**Reply by rohitgeddam** [reply]
freenode.net#machomebrew (IRC)

https://github.com/Homebrew/brew in README community section

**Reply by rohitgeddam** [reply]
We should remove IRC mention if we are not using it actively. can I make that small change in README.md

**Reply by Rylan12** [reply]
Feel free to open a PR!

**Reply by rohitgeddam** [reply]
Sure, Thanks
I have opened a - [PR](https://github.com/Homebrew/brew/pull/8981)
The CI tests are failing even though I have just made a small change in README.md

**Reply by kode54** [reply]
Gee, someone should have told me that this was the case. I wondered why known brew maintainers refused to do anything but idle 24/7/365 in the IRC channel. I guess it was also too much to expect that #homebrew be for an actual open source software project, and not brewing alcohol. I guess I should also expect Liberachat to have open source cheese fermenting discussions too.

**Reply by Rylan12** [reply]
@kode54 please read our [code of conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md) and refrain from commenting when you have nothing useful or relevant to add.
